PiwikTracker/PiwikUserDefaults.swift

```diff
@@ -0,0 +1,94 @@
+import Foundation
+
+/// PiwikUserDefaults is a wrapper for the UserDefaults with properties
+/// mapping onto values stored in the UserDefaults.
+/// All getter and setter are sideeffect free and automatically syncronize
+/// after writing.
+internal class PiwikUserDefaults {
```

Comment:
I recommend to always mark classes as `final` unless they should be subclasses (performance reason)